cmd/bricksllm/.env

```diff
@@ -1,4 +1,4 @@
-CONFIG_FILE_NAME=
+CONFIG_FILE_NAME=./config_local.json
```

Comment:
when you set this, the default application behavior is using variables from config_local.json. can we set this to empty so we don't change the default behavior?

Reply:
Done.